.gitignore

```diff
@@ -68,3 +68,6 @@ Tests/cucumber/Gemfile*
 
 # Gemfile used in rspec tests for CLI
 Gemfile.lock
+#
+.bundle
+vendor
```

Comment:
Why do we need these changes?

I would prefer these were in a different pull request.